docs/units/format.rst

```diff
@@ -188,15 +188,19 @@ following formats:
       2.1798721×10-¹⁸ ─────
                        s²
 
-Unrecognized Units
-==================
+.. _astropy-units-format-unrecognized:
+
+Dealing with Unrecognized Units
+===============================
 
 Since many files found in the wild have unit strings that do not
 correspond to any given standard, `astropy.units` also has a
 consistent way to store and pass around unit strings that did not
-parse.
+parse.  In addition, it provides tools for transforming non-standard,
+legacy or misspelt unit strings into their standardised form,
```

Comment:
```suggestion
legacy or misspelt unit strings into their standardized form,
```
(for consistency, we use US spelling in the documentation https://docs.astropy.org/en/stable/development/style-guide.html#spelling)

Reply:
OK, done.